### DISCUSSION TITLE
Rebuilding ExtensionBuilder.json

### DISCUSSION THREAD

**Discussion by rl-saen** [FriendsOfTYPO3/extension_builder#332] [Q&A] [answered]
Hello everyone,
I have recently taken over support of a client's typo3 project. The main extension was built with ExtensionBuilder but for whatever reason the `ExtensionBuilder.json` file is missing, so I can't open the extension in the backend module.

Is there a way to restore it other than doing it manually?

**Comment by liayn** [comment]
I don't know a way to achieve this other than manual. Sorry.

**Comment by DavidBruchmann** [comment, marked as answer]
usually the file `ExtensionBuilder.json` is removed intentionally and there is no way to recover it if you never have a copy or link to a repository. Question is if it's worth it to rebuild it or if you just can add a few things manually.
If you want to add a lot like new tables you can create only the new parts in extension_builder and merge the result manually with the existing extension.

**Reply by rl-saen** [reply]
Well, its only four models, so I guess I'll just create a new extensions and remodel them there then copy it over.

**Reply by DavidBruchmann** [reply]
It's advisable to create the new extension with the same key and names.
Else you've much work to rename the things.
